src/components/pagination/Pagination.tsx

```diff
@@ -0,0 +1,36 @@
+import { useState } from 'react';
+import './pagination.scss'; 
```

Comment:
With scss modules this should be 

```suggestion
import styles './pagination.scss'; 
```